client/signup/steps/website-content/section-generator.tsx

```diff
@@ -1,14 +1,14 @@
+import { TranslateResult } from 'i18n-calypso';
 import {
 	AccordionSectionProps,
 	SectionGeneratorReturnType,
 } from 'calypso/signup/accordion-form/types';
 import { WebsiteContent } from 'calypso/state/signup/steps/website-content/schema';
 import { CONTENT_SUFFIX, PageDetails } from './page-details';
 
-const PAGE_TITLES: Record< string, string > = { Home: 'Homepage' };
-
 export const sectionGenerator = ( params: SectionGeneratorReturnType< WebsiteContent > ) => {
 	const { translate, formValues, formErrors, onChangeField } = params;
+	const PAGE_TITLES: Record< string, TranslateResult > = { Home: translate( 'Homepage' ) };
```

Comment:
Since, above, Homepage was changed to home should we also do that here?

Reply:
Oh yes, I missed this, good catch!